test/packages/custom_logs/agent/stream/input.yml.hbs

```diff
@@ -0,0 +1,12 @@
+paths:
+{{#each paths}}
+  - {{this}}
+{{/each}}
+
+tags:
+{{#each tags}}
+  - {{this}}
+{{/each}}
+
```

Comment:
I think there is a few minimum additions to this, that needs to be in every input package, and many of them are in other integrations as well:

Pipelines:
```
{{#if pipeline}}
pipeline: {{pipeline}}
{{/if}}
```

Tags are slightly wrong, they should be like this:

```
{{#if tags}}
tags:
{{#each tags as |tag i|}}
  - {{tag}}
{{/each}}
{{/if}}
```

Since an input can ingest data both from a local host and external host, we need to add support for removing the host fields:

```
{{#contains "forwarded" tags}}
publisher_pipeline.disable_host: true
{{/contains}}
```

And custom processors:
```
{{#if processors}}
processors:
{{processors}}
{{/if}}
```

Reply:
Custom processing has been intentionally left out of the MVP, it would also require custom mappings.

Also, depending on how this is implemented, this wouldn't require anything on the spec, the pipelines and the mappings could be configured in Fleet and installed by it.

Reply:
I added a few of your ideas to the sample package.